kubernetes/samples/scripts/create-soa-domain/wlst/create-domain-script.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Is this pretty much the same as the script in create-fmw-infrastructure-domain ?